### ISSUE TITLE
ботстрат

### ISSUE THREAD

**Issue by arthurash** [ogamespec/ogame-opensource#70]
Привет, я не понимаю, как работает бот-система?
Это работает?
у вас есть пример ботстрата?

**Comment by ogamespec** [comment]
Привет, подтягиваются наши))
Надо припомнить всё, я недавно тут порядок навёл и запустил тестовый сервер на http://ogamewtf.com
Пока есть проблемы с переходом на пыху 7.x, вначале их хочу устраниться.

По ботстратам - там вроде тривиально, но я не уверен что BotAPI покрывает все возможности движка.. Постараюсь написать мануал какой-то.

**Comment by ogamespec** [comment]
Хороший вопрос. Вспомнить бы ещё как их эскпортировать/импортировать :-)

**Comment by arthurash** [comment]
прямо из базы данных botstrat

**Comment by ogamespec** [comment]
Нет, такого под рукой нет к сожалению. Но в целом там применяется ДРАКОН схемы. То есть одна общая шина и "шампуры" с логикой.

**Comment by arthurash** [comment]
Все еще нет решения для ботов?

**Comment by ogamespec** [comment]
Пока не нашёл время и вдохновения позаниматься проектом :)

**Comment by ogamespec** [comment]
Нашёл старую документацию на ботов, там есть немного информации:

https://github.com/ogamespec/ogame-opensource/blob/master/Wiki/ru/ai.md

Но очевидно, надо ещё развивать эту подсистему.

**Comment by ogamespec** [comment]
Закрыл этот итем, т.к. не похоже на какую-то ошибку. Обсудить остальное можно в дискорде или разделе Discussions.
